client/shaders/blur_v/opengl_fragment.glsl

```diff
@@ -0,0 +1,29 @@
+#define rendered texture0
+
+uniform sampler2D rendered;
+uniform vec2 texelSize0;
+uniform lowp float bloomRadius = 3.0;
+
+#ifdef GL_ES
+varying mediump vec2 varTexCoord;
+#else
+centroid varying vec2 varTexCoord;
+#endif
+
+void main(void)
```

Comment:
I assume there's a good reason. Just for my personal education, why handle horizontal and vertical blur separately instead of in one go?

Reply:
Computational complexity is O(r) instead of O(r²) where r is the blur radius / distance.